airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -1343,6 +1346,9 @@
   icon: postgresql.svg
   sourceType: database
   releaseStage: generally_available
+  allowedHosts:
+    hosts:
+      - "${host}"
```

Comment:
@git-phu I'd love your 👍 on how I ended up enabling interpolation.  By the time we get to the KubeProcessFactory (or DockerProcessFactory), these strings are all resolved, but another pair of 👀 can't hurt.